gradle/gradle-mvn-mpp-push.gradle

```diff
@@ -26,6 +26,14 @@ def getRepositoryPassword() {
   return hasProperty('SONATYPE_NEXUS_PASSWORD') ? SONATYPE_NEXUS_PASSWORD : ""
 }
 
+def hasInMemorySigningKey() {
+  return hasProperty('PGP_SIGNING_KEY')
+}
+
+def getInMemorySigningKey() {
+  return hasProperty('PGP_SIGNING_KEY') ? PGP_SIGNING_KEY.replace("\\n", "\n") : ""
```

Comment:
The replace here is to work around the fact that Azure can only pass single-line variables. Need to read the pgp block as a multi-line string.

You can also pass a multi-line string in gradle.properties if you want to do that locally, but note that it needs a `\n\` at the end of every line.